### DISCUSSION TITLE
Changing the title of " chapter two" in the Overleaf shows an error

### DISCUSSION THREAD

**Discussion by GIGASHAB** [joaomlourenco/novathesis#269] [Q&A]
Hi,

 Changing the title of " chapter two" in the Overleaf shows an error in compiling to pdf. While it doesn´t have any problem in other chapters. I would be thankful if you let me know what is the problem.

![Overleaf](https://user-images.githubusercontent.com/96875437/148051182-243977e8-5e12-4434-9241-83bd06053ff7.png)

note: I used both version: joaomlourenco-novathesis-6.8.6-20-gc217e0e (downloaded from Git) and NOVAthesis v6.5.3.

Regards

**Comment by joaomlourenco** [comment]
You have to select the file "tempalte.tex" as the main file (not chapter2.tex).

**Comment by GIGASHAB** [comment]
Yes, But is it Ok by compiling Chapter(2), that error appears?

**Reply by joaomlourenco** [reply]
Dear @GIGASHAB

You cannot use the template if you do not understand the basic concepts of LaTeX.  So, please, take care of learning some LaTeX first.  There is a good introduction named "A not so short introduction to LaTeX".

Them, I strongly suggest that you find a colleague that has used the template and ask him/her for some initial guidance.  That kind of support/guidance cannot be given here.

Thank you.
